.github/workflows/ci.yml

```diff
@@ -34,12 +34,8 @@ jobs:
       - run: yarn lint
 
   basic-tests:
-    name: "Basic Tests - ${{ matrix.os }}"
-    runs-on: "${{ matrix.os }}-latest"
-
-    strategy:
-      matrix:
-        os: [ubuntu, macOS, windows]
```

Comment:
All basic tests are still run on `ubuntu` and `windows` in the `tests` job below via the `test:all` script.